daphne_worker/src/config.rs

```diff
@@ -121,10 +121,6 @@ pub(crate) struct DaphneWorkerConfig {
     /// configured by the Leader.
     pub(crate) helper_state_store_garbage_collect_after_secs: Option<Duration>,
 
```

Comment:
Just a note that we'll want to reimplement something like this for the AggregateStore DO at some point (https://github.com/cloudflare/daphne/issues/413).